Source/ThirdParty/TurboBadger/tb_widgets.cpp

```diff
@@ -359,7 +359,8 @@ void TBWidget::AddChildRelative(TBWidget *child, WIDGET_Z_REL z, TBWidget *refer
 
 void TBWidget::RemoveChild(TBWidget *child, WIDGET_INVOKE_INFO info)
 {
-    assert(child->m_parent);
+    if (!child->m_parent)
+        return;
 
```

Comment:
Ok, here's the crash I am seeing on macOS in release mode, I tried a few times in debug and didn't trip it:

https://github.com/AtomicGameEngine/AtomicGameEngine/issues/1216

I haven't verified tried to reproduce on Windows yet